SetFlags.cmake

```diff
@@ -86,7 +86,11 @@ macro(set_flags)
 		else()
 			add_flags_cxx("-pthread")
 		endif()
-
+	elseif(ANDROID)
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
Then let me advise you to stick to c++11 to avoid having a similar problem to when a PR wouldn't compile on windows because msvc exposes std::make_unique, which was causing issues with my make_unique if it was in the global namespace.